### PR TITLE
Update Sign and Submit screens

### PR DESCRIPTION
**Pull request by vrajmohan**

### CODE REVIEW

File: src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "IsViolatingParole",
+            "ConvictedForLyingAboutBenefits",
+            "ConvictedForTradingBenefits",
+            "IsReceivingBenefitsWithFakeID"
+    };
+
+    @Override
+    public void run(Submission submission) {
+        Arrays.stream(SENSITIVE_CONVICTION_QUESTIONS).forEach(s -> submission.getInputData().putIfAbsent("noOne" + s + "[]", "true"));
+    }
```
Comment (bseeger) [inline src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java:31]: Why are you creating a list like name here when the value isn't a list?  Maybe you don't need the "[]"?

File: src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java [hunk 2]
```diff
@@ -36,6 +39,20 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
         results.put("applicantSSN", new SingleField("applicantSSN", SubmissionUtilities.formatSSN((String) inputData.get("encryptedSSN")), null));
         results.put("speaksEnglish", new SingleField("speaksEnglish", (String) "Yes", null));
 
+        prepareAnswersToSensitiveQuestions(inputData, results);
         return results;
     }
+
+    private static void prepareAnswersToSensitiveQuestions(Map<String, Object> inputData, Map<String, SubmissionField> results) {
+        for (String question : SetDefaultSensitiveConvictionAnswers.SENSITIVE_CONVICTION_QUESTIONS) {
```
Comment (bseeger) [inline src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java:47]: Pulling these from a specific Action feels weird, but maybe having as an enum feels weird as well.

File: src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java (hunk 1 above)
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java:31]: You'd only use the `[]` if the original var looked like this: 
`private List<String> noOneHasDrugKingpinFelony;`

Or at least that's what we've used the brackets to indicate so far.

File: src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+}
```
Comment (bseeger) [inline src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java:32]: Adding a test for this class would be useful.

File: src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java (hunk 2 above)
Comment (vrajmohan) [reply src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java:46]: Agreed, I couldn't think of a proper place to put it.

File: src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java (hunk 1 above)
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java:20]: We talked through this and I understand more now.  Resolving this.

**Review by bseeger — APPROVED** [review]
LGTM - thanks for the changes.

**Pull request merged**